CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 # Changelog
 
-## v0.20.x - TBD
+## v0.21.x - TBD
+
+## v0.20.x - 2020-02
+
+* feat: add new support for testing with environment variables (#180)
+* feat: logging can be enabled via env var (#181)
+* refactor: copy IsProtoEqual from spanner (#179)
+* chore: bump version numbers after v0.19 release (#174)
```

Comment:
Probably want to drop the chore about bumping the version numbers.

Reply:
Done.  Thanks.